### PR TITLE
grafana doc

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

File: content/getting-started/canary.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ kubectl delete namespace $NAMESPACE
 
 ### Inspect progress using Grafana
 
-Coming soon
+You can inspect the progress of your experiment using the sample *iter8 Metrics* dashboard. To install this dashboard, see [here](../integrations/grafana/).
```
Comment (Alan-Cha) [inline content/getting-started/canary.md:197]: This link does not work

Comment (Alan-Cha) [reply content/getting-started/canary.md:197]: Should it be

```
[here](../../integrations/grafana/)
```

Comment (Alan-Cha) [reply content/getting-started/canary.md:197]: Here is the deploy preview link: https://deploy-preview-210--focused-benz-34dc64.netlify.app/getting-started/canary/

Comment (kalantar) [reply content/getting-started/canary.md]: @Alan-Cha thanks for catching; fixed. How do I figure out the preview link?

**Pull request merged**